navigation/navigation.css

```diff
@@ -0,0 +1,32 @@
+.navigation-heading{
+    display: flex;
+    justify-content: center;
+}
+
+.navigation-container{
+    display: flex;
+    background-color:aquamarine;
```

Comment:
If you are going to use this 'aquamarine' color anywhere else as well then try to assign color in variable and then use it wherever you want. Rest seems good.